pkg/services/contexthandler/contexthandler.go

```diff
@@ -23,22 +24,21 @@ import (
 	"github.com/grafana/grafana/pkg/web"
 )
 
-func ProvideService(cfg *setting.Cfg, tracer tracing.Tracer, features featuremgmt.FeatureToggles, authnService authn.Service,
```

Comment:
@kalleep I forgot to check in with you, but is it okay to remove the dependency on FeatureToggles, they were not in use in the contexthandler and removing it would make my using it easier in the standalone code.